frontend/src/tbpro/elements/TextInput.vue

```diff
@@ -1,6 +1,18 @@
 <script setup>
 import { ref } from 'vue';
 
+const inputRef = ref(null);
+/**
+ * Forwards focus intent to the text input element.
+ * Unlike HTMLElement.focus() this does not take any parameters.
+ */
+const focus = () => {
+  if (!inputRef.value) {
+    return;
+  }
+  inputRef.value.focus();
+};
```

Comment:
In order to focus the input I need to forward the focus intent to the input element. Pretend this is HTMLElement.focus but without the options param.